src/frontend/src/components/Menu.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+import { useLocation } from "react-router";
+
+import Button from "./base/Button/Button";
+import { NavTab } from "./base/NavTab/NavTab";
+import {
+    Dropdown,
+    DropdownItem,
+    DropdownTrigger,
+} from "./base/Dropdown/Dropdown";
+import useMediaQuery from "hooks/useMediaQuery";
+
+import EchoTextLogo from "assets/icons/echo-logo-text.svg?react";
+import EchoLogo from "assets/icons/echo-logo.svg?react";
+import HamburgerIcon from "assets/icons/hamburger.svg?react";
+import ArrowDownIcon from "assets/icons/arrow-down.svg?react";
+
+interface Props {
+    compareCount: number;
+}
+
+const Menu = ({ compareCount }: Props) => {
+    // TODO: hook up with app-wide language state
+    const [selectedLanguage, setSelectedLanguage] = useState("en");
+    const { pathname } = useLocation();
+    const isDesktop = useMediaQuery("(min-width: 768px)");
+
+    const languages: { [key: string]: string } = {
+        en: "EN",
+        es: "ES",
+        zh: "中文",
+    };
+
+    return (
+        <div className="flex w-full h-9 justify-between items-center gap-5 self-stretch flex-shrink-0 py-0 pl-4 pr-3">
+            {/* The ECHO icon section */}
+            <div className="flex items-center gap-0 flex-1 pt-2 pb-3 pl-3">
+                {isDesktop ? (
+                    <EchoTextLogo className="h-[19px]" />
+                ) : (
+                    <EchoLogo className="h-[19px] w-[20px]" />
+                )}
+            </div>
+
+            {/* The Discover/Compare section */}
+            {(pathname.includes("discover") ||
+                pathname.includes("compare")) && (
+                <div className="flex self-stretch justify-center gap-2">
```

Comment:
```suggestion
            {/* The Discover/Compare section */}
            {(
                <div className="flex self-stretch justify-center gap-2">
```

Can we remove this conditional? I think we would want to include these tabs regardless of the page, even if neither of them display as "actively" selected

Reply:
We could remove the conditional, but I would not recommend it. 

For example, the following sign-in page does not have those two tabs, which will likely be under `/login` or just `/`. Since we want to show pages and components based on app paths, the conditions in the component right now serve as a starting point. As we build more pages under more paths, this conditional section and other menu sections will change based on path or app state. So I recommend we leave it as is, and we iterate from here.

![Screenshot 2025-07-08 at 5 51 32 PM](https://github.com/user-attachments/assets/81f06c23-de75-4540-92b4-3a5d44dcd38f)

Reply:
That makes sense, ok let's leave it in 👍